examples/conda/README.md

```diff
@@ -0,0 +1,17 @@
+# Build a Tesseract from a Conda environment
+
+To build a Tesseract from a
```

Comment:
I don't know whether there should be a or an here 🤔

Reply:
pretty sure it's "an"